docs/blog/newsletter-2018-august.rst

```diff
@@ -0,0 +1,109 @@
+
+Next month's issue of the newsletter will be my last as editor. Since we started the newsletter just before the 2016 Prague conference, it's been my privilege to work with our dedicated team of volunteers to put together these digests from the documentation community every month. I'm sharing all of this both to thank you all for reading over these last two years, but also to put out the call for a new editor! It's a wonderful way to give back to the Write the Docs community and you couldn't ask for a better group of folks to work alongside.
+
+If you'd be interested in learning more, drop me an email at newsletter@writethedocs.org or ping me on Slack (@kobrien042) and I'll be happy to tell you more!
+
+In the meantime, let's tune into some highlights from the hiatus!
+
+-----------------------------------------
+Common misconceptions about documentation
+-----------------------------------------
+
+Inspired by `Emily January Petersen's guest post <http://idratherbewriting.com/2018/07/18/stereotypes-about-tech-writers-in-workplace/>`_ on `I'd Rather Be Writing <http://idratherbewriting.com>`_, documentarians discussed some common misconceptions about our profession that they've personally experienced. As one person pointed out, **we** know that it takes more than being able to use a word processor or a text editor to be a documentarian... but what about everyone else?
+
+Here are some of the unfortunately common misconceptions mentioned in Slack this month:
+
+- Anyone can write good docs.
+- No one reads the docs.
+- Documentation is fast and easy to produce.
+- Documentation doesn't have any discernible business value.
+- Tech writers are just there to make docs pretty.
+- It's not a role that earns much respect or recognition.
+
+Having this conversation with a bunch of other documentarians meant there was no shortage of support and commiseration to be found. When it comes to showing other people the error of their ways, one practical suggestion was to lean on whatever analytics or reporting you have – and if you don't have any, start collecting some. Another, perhaps less serious suggestion was to just stop writing the docs for a month and see how things go then. ;)
+
+
+*The* `#career-advice <https://writethedocs.slack.com/messages/career-advice>`_ *channel is always full of sympathetic ears and practical advice for whatever professional frustrations you may encounter.*
+
+---------------------------------------------------------------------------
+In the time of web-based applications, what is a page and what is a screen?
+---------------------------------------------------------------------------
+
+An classic question of word choice arose this month: What's the difference between a "page", a "screen" – and which term should be used when?
```

Comment:
What's the difference between a page and a screen - and which term should be used when?